### PR TITLE
[VI] Add `Beatmapping/Hitsound`

### PR DESCRIPTION
**Pull request by Datcty**
I translated this article into Vietnamese.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by Walavouchey** [comment]
hi ty for contributing, just noting that you're supposed to make a new `vi.md` file in the same directory

**Comment by Pisapou** [comment]
Also, just for readability, keep the same formatting of title as other pr: [LANG] Add/Update `Article`.
So for your article, it would be: [VI] Add `Beatmapping/Hitsound`

**Comment by Datcty** [comment]
Sorry, I'm pretty new to GitHub.

**Review by Datcty — COMMENTED** [review]
I hope this is good. I want it to be a part of the osu!wiki.

**Comment by TranT3011** [comment]
> I hope this is good. I want it to be a part of the osu!wiki.
i dont think you can actualy review yourself lol

File: wiki/Beatmapping/Hitsound/vi.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+stub: true
+tags:
+  - hit normal
+  - hitnormal
+  - hit sound addition
+  - hit sound additions
+  - hitsound addition
+  - hitsound additions
+  - hit sounds
+  - hitsounds
```
Comment (Walavouchey) [inline wiki/Beatmapping/Hitsound/vi.md:11]: these tags need to be translated (while also keeping the originals). they help when searching for the article

Comment (Datcty) [reply wiki/Beatmapping/Hitsound/vi.md:11]: yes sure, I will translate them

File: wiki/Beatmapping/Hitsound/vi.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ tags:
   - hitsound additions
   - hit sounds
   - hitsounds
+  - âm thanh
```
Comment (TranT3011) [inline wiki/Beatmapping/Hitsound/vi.md:12]: delete this because hitsound cant really be translated to vietnamese

**Review by TranT3011 — CHANGES_REQUESTED** [review]

File: wiki/Beatmapping/Hitsound/vi.md (hunk 2 above)
Comment (Datcty) [reply wiki/Beatmapping/Hitsound/vi.md:12]: ok thanks

Comment (TranT3011) [reply wiki/Beatmapping/Hitsound/vi.md]: um actually i do have some other problems to fix so plz wait me

**Comment by Walavouchey** [comment]
reminder to name your commits rather than leaving them at default, to make it easier to know what they do at a glance



more info in the [contribution guide](https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#applying-reviews) (images are dead atm)

File: wiki/Beatmapping/Hitsound/vi.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+---
+stub: true
+tags:
+  - hit normal
+  - hitnormal
+  - hit sound addition
+  - hit sound additions
+  - hitsound addition
+  - hitsound additions
```
Comment (TranT3011) [inline wiki/Beatmapping/Hitsound/vi.md:9]:
```suggestion
  - hit sound bổ sung
  - hitsound bổ sung
```

File: wiki/Beatmapping/Hitsound/vi.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  - hit sounds
+  - hitsounds
+---
+
+# Âm thanh
```
Comment (TranT3011) [inline wiki/Beatmapping/Hitsound/vi.md:14]:
```suggestion
# Hitsound
```
cant be translated to vnese

File: wiki/Beatmapping/Hitsound/vi.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+*Xem: [Thêm âm thanh tuỳ chỉnh](/wiki/Guides/Using_custom_hitsounds)*
```
Comment (TranT3011) [inline wiki/Beatmapping/Hitsound/vi.md:16]:
```suggestion
*Xem: [Thêm hitsound tuỳ chỉnh](/wiki/Guides/Using_custom_hitsounds)*
```

File: wiki/Beatmapping/Hitsound/vi.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+<!-- TODO: could make more articles about each part of hitsounds, and also one focused more on gameplay than mapping -->
+
+**Âm thanh** là những âm thanh osu! phát ra để đáp lại khi người dùng tương tác với  [vật thể](/wiki/Hit_object). Âm thanh được dùng để tạo ra sự phản hồi cho người chơi để giúp họ cải thiện [độ chính xác](/wiki/Gameplay/Accuracy) trong bài hát.
```
Comment (TranT3011) [inline wiki/Beatmapping/Hitsound/vi.md:20]:
```suggestion
**Hitsound** là những âm thanh osu! phát ra để đáp lại khi người dùng tương tác với [vật thể](/wiki/Hit_object). Âm thanh được dùng để tạo ra sự phản hồi cho người chơi để giúp họ cải thiện [độ chính xác](/wiki/Gameplay/Accuracy) trong bài hát.
```

File: wiki/Beatmapping/Hitsound/vi.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+Âm thanh bao gồm một mẫu mặc định được gọi là *hitnormal* và bất kỳ sự kết hợp nào của *mẫu bổ sung* còi (whistle), kết thúc (finish) hoặc vỗ tay (clap). Mỗi mẫu là một phần của một trong ba *bộ mẫu* biểu thị các kiểu khác nhau: `Bình thường`, `Nhẹ` hoặc `Trống`.
```
Comment (TranT3011) [inline wiki/Beatmapping/Hitsound/vi.md:22]:
```suggestion
Hitsound được phát ra một mẫu mặc định gọi là *hitnormal* và sự kết hợp của *mẫu bổ sung* còi (whistle), kết thúc (finish) hoặc vỗ tay (clap). Mỗi mẫu trên là một phần của một trong ba *độ mạnh* phát ra các kiểu khác nhau: `Normal` (bình thường), `Soft` (nhẹ), hoặc `Drum` (trống).
```
i made this sound more native

File: wiki/Beatmapping/Hitsound/vi.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+Mapper có thể đổi chỗ bất kỳ mẫu mặc định nào với một mẫu âm thanh tuỳ chỉnh bằng cách sử dụng thư mục của [biểu đồ](/wiki/Beatmap). [Skin](/wiki/Skinning) của người chơi cũng có thể thay đổi âm thanh mặc định của tất cả biểu đồ. Chi tiết về các mẫu âm thanh có thể được tìm thấy ở bài viết [Skin: Âm thanh](/wiki/Skinning/Sounds#hitsounds).
```
Comment (TranT3011) [inline wiki/Beatmapping/Hitsound/vi.md:24]:
```suggestion
Mapper có thể thay đổi bất kỳ hitsound mặc định nào với những âm thanh khác tuỳ chỉnh bằng cách sử dụng thư mục của [biểu đồ](/wiki/Beatmap). [Skin](/wiki/Skinning) của người chơi cũng có thể thay đổi hitsound của tất cả biểu đồ. Chi tiết về các hitsound có thể được tìm thấy ở bài viết [Skin: Hitsounds](/wiki/Skinning/Sounds#hitsounds).
```
same as 22

File: wiki/Beatmapping/Hitsound/vi.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+[Vòng xoay](/wiki/Hit_object/Spinner) và [thanh trượt](/wiki/Hit_object/Slider) có những âm thanh của riêng chúng: spinner spin, spinner bonus, slider slide, và slider tick.
```
Comment (TranT3011) [inline wiki/Beatmapping/Hitsound/vi.md:26]:
```suggestion
[Vòng xoay](/wiki/Hit_object/Spinner) và [thanh trượt](/wiki/Hit_object/Slider) có những hitsound của riêng chúng: spinner spin, spinner bonus, slider slide, và slider tick.
```

File: wiki/Beatmapping/Hitsound/vi.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+
+## Âm thanh chủ động
```
Comment (TranT3011) [inline wiki/Beatmapping/Hitsound/vi.md:28]:
```suggestion
## Hitsound chủ động
```

File: wiki/Beatmapping/Hitsound/vi.md [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+
+Một âm thanh được gọi là *chủ động* nếu nó được đặt ở một vật thể tương tác được, chẳng hạn như vòng tròn hoặc đầu của thanh trượt. Những âm thanh này cho người chơi biết độ chính xác của họ bằng cách dùng một âm thanh to rõ vào lúc họ nhấp chuột. Nếu được nhấp chuột đúng lúc, âm thanh sẽ trở nên phù hợp với nhịp điệu của bài hát.
```
Comment (TranT3011) [inline wiki/Beatmapping/Hitsound/vi.md:30]:
```suggestion
Một hitsound được gọi là *chủ động* nếu nó được đặt ở một vật thể tương tác được (nốt), chẳng hạn như vòng tròn hoặc đầu của thanh trượt. Những hitsound này cho người chơi biết độ chính xác của họ bằng cách dùng một âm thanh to rõ vào lúc họ nhấp chuột. Nếu được nhấp chuột đúng lúc, âm thanh sẽ trở nên phù hợp với nhịp điệu của bài hát.
```
same as 22

File: wiki/Beatmapping/Hitsound/vi.md [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+
+## Âm thanh bị động
```
Comment (TranT3011) [inline wiki/Beatmapping/Hitsound/vi.md:32]:
```suggestion
## Hitsound bị động
```

File: wiki/Beatmapping/Hitsound/vi.md [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+
+Một âm thanh được gọi là *bị động* nếu nó được đặt ở phần không tương tác được của vật thể, chẳng hạn như thanh trượt lặp lại hoặc đuôi của vòng xoay. Chúng luôn phát ra miễn là người chơi tương tác với vật thể, không giống như các âm thanh chủ động. Các âm thanh bị động không cần phải rõ ràng hoặc có thể nghe được, nhưng thường sử dụng giống như các âm thanh chủ động khi chúng được nhấp theo các nhịp riêng biệt để giúp người chơi duy trì nhịp điệu.
```
Comment (TranT3011) [inline wiki/Beatmapping/Hitsound/vi.md:34]:
```suggestion
Một hitsound được gọi là *bị động* nếu nó được đặt ở phần không tương tác được của vật thể, chẳng hạn như thanh trượt lặp lại hoặc đuôi của vòng xoay. Chúng luôn phát ra âm thanh miễn là người chơi tương tác với vật thể, không giống như các hitsound chủ động. Các hitsound bị động không cần phải rõ ràng hoặc có thể nghe được, nhưng thường sử dụng giống như các hitsound chủ động khi chúng đi theo các nhịp riêng biệt để giúp người chơi duy trì nhịp điệu.
```
same as 22

File: wiki/Beatmapping/Hitsound/vi.md [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+
+## Âm thanh Storyboard
```
Comment (TranT3011) [inline wiki/Beatmapping/Hitsound/vi.md:36]:
```suggestion
## Hitsound storyboard
```

File: wiki/Beatmapping/Hitsound/vi.md [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+
+Các mẫu âm thanh có thể được thêm vào thông qua Storyboard để bắt chước các âm thanh. Tuy nhiên, không giống như những âm thanh thông thường, những âm thanh này luôn phát vào thời điểm nhất định bất kể người chơi có nhấn trúng đối tượng hay không. Vì lý do đó, chúng thường không được khuyến khích. Một số mapper sử dụng chúng để duy trì các mẫu âm thanh giữa các vật thể.
```
Comment (TranT3011) [inline wiki/Beatmapping/Hitsound/vi.md:38]:
```suggestion
Các mẫu âm thanh có thể được thêm vào thông qua Storyboard để bắt chước các hitsound. Tuy nhiên, không giống như những hitsound thông thường, chúng luôn phát vào thời điểm nhất định kể cả người chơi có nhấn trúng vật thể hay không. Vì lý do đó, chúng thường không được khuyến khích. Một số mapper sử dụng chúng để duy trì các mẫu âm thanh giữa các vật thể.
```
same as 22

File: wiki/Beatmapping/Hitsound/vi.md [hunk 16]
```diff
@@ -0,0 +1,48 @@
...
+
+## Keysound
+
+Một "keysound" là một mẫu âm thanh mà cực kỳ giống với, hoặc được lấy trực tiếp từ, bài hát và được dùng để mô phỏng theo những nốt của bài hát. Cách đặt âm thanh này thường cung cấp sự phản hồi kém cho người chơi, nhưng có thể làm cho trải nghiệm chơi vui hơn và làm cho các phần của biểu đồ trở nên nổi bật.
```
Comment (TranT3011) [inline wiki/Beatmapping/Hitsound/vi.md:42]:
```suggestion
Một "keysound" là một mẫu hitsound mà cực kỳ giống với, hoặc được lấy trực tiếp từ bài hát và được dùng để mô phỏng theo những nốt của bài hát. Cách đặt hitsound này thường cung cấp sự phản hồi kém cho người chơi, nhưng có thể làm cho trải nghiệm chơi vui hơn và làm cho các phần của biểu đồ trở nên nổi bật.
```

File: wiki/Beatmapping/Hitsound/vi.md [hunk 17]
```diff
@@ -0,0 +1,48 @@
...
+
+## Với osu!taiko
+
+Không giống như những [chế độ chơi](/wiki/Game_mode) khác, âm thanh của [osu!taiko](/wiki/Game_mode/osu!taiko) ảnh hưởng trực tiếp đến gameplay của biểu đồ. Kat được phân biệt với Don bởi tiếng còi (whistle) và vỗ tay (claps), và những nốt lớn được phân biệt với những nốt bình thường bởi những tiếng kết thúc (finish).
```
Comment (TranT3011) [inline wiki/Beatmapping/Hitsound/vi.md:46]:
```suggestion
Không giống như những [chế độ chơi](/wiki/Game_mode) khác, hitsound của [osu!taiko](/wiki/Game_mode/osu!taiko) ảnh hưởng trực tiếp đến gameplay của biểu đồ. Kat được phân biệt với Don bởi tiếng còi (whistle) và vỗ tay (claps), và những nốt lớn được phân biệt với những nốt bình thường bởi những tiếng kết thúc (finish).
```

File: wiki/Beatmapping/Hitsound/vi.md [hunk 18]
```diff
@@ -0,0 +1,48 @@
...
+
+Những mẫu âm thanh mặc định của osu!taiko thì độc nhất, và mapper thường không thay đổi chúng.
```
Comment (TranT3011) [inline wiki/Beatmapping/Hitsound/vi.md:48]:
```suggestion
Những mẫu hitsound mặc định của osu!taiko độc nhất, và mapper thường không thay đổi chúng.
```
same as 22

**Review by TranT3011 — CHANGES_REQUESTED** [review]
if you agree then just click add suggestion to batch
excuse me for writting the older suggestion, i was supposed to suggest in file changed not commits

File: wiki/Beatmapping/Hitsound/vi.md [hunk 19]
```diff
@@ -0,0 +1,46 @@
...
+tags:
+  - hit normal
+  - hitnormal
+  - hit sound bổ sung
+  - hitsound bổ sung
+  - hit sounds
+  - hitsounds
+---
+
+# Âm thanh
```
Comment (TranT3011) [inline wiki/Beatmapping/Hitsound/vi.md:12]:
```suggestion
# Hitsound
```

**Review by TranT3011 — CHANGES_REQUESTED** [review]
also there are suggestions that you havent applied/given me reason why you dont want to change

**Comment by TranT3011** [comment]
> also there are suggestions that you havent applied/given me reason why you dont want to change

any suggestion that isnt "outdated" means it hasnt been touched

**Pull request merged**